affiliates/estonia.swagger.json

```diff
@@ -0,0 +1,515 @@
+        } ]
+      }
+    }
+  },
+  "components" : {
+    "securitySchemes": {
+      "oAuth": {
+        "type": "oauth2",
+        "flows": {
+          "clientCredentials": {
+            "tokenUrl": "/token",
+            "scopes": { 
+              "allow_custom_pingtree": ""
+            }
+          }
+        }
+      },
+      "bearerAuth": {
+        "type": "http",
+        "scheme": "bearer"
+      }
+    },
+    "schemas" : {
+      "EmployerAddress" : {
+        "type" : "object",
+        "properties" : {
+          "city" : {
+            "type" : "string",
+            "example" : "Tartu",
+            "description" : "Required if product === 'BUSINESS', Format: /^([a-zA-ZÄÖÜÕŽŠäöüõžš]+[\\s\\-\\']?)+$/ui",
+            "maxLength": 45
+          },
+          "street" : {
+            "type" : "string",
+            "example" : "Riia",
+            "description" : "Required if product === 'BUSINESS', Format: /^([a-zA-ZÄÖÜÕŽŠäöüõžš]+[\\s\\-\\']?)+$/ui",
+            "maxLength": 45
+          },
+          "house_number" : {
+            "type" : "string",
+            "example" : "1",
+            "description" : "Required if product === 'BUSINESS', Format: /^[0-9]+[\\-]?[a-zA-Z]?$/u",
+            "maxLength": 45
+          },
+          "flat_number" : {
+            "type" : "string",
+            "example" : "4",
+            "description" : "Format: /^[0-9]+[\\-]?[a-zA-Z]?$/u",
+            "maxLength": 255
+          }
+        }
+      },
+      "Agreements" : {
+        "type" : "object",
+        "additionalProperties" : {
+          "$ref" : "#/components/schemas/Agreement"
+        },
+        "example" : {
+          "terms_of_service" : true,
+          "data_proccessing_policy" : true
+        }
+      },
+      "Agreement" : {
+        "type" : "boolean"
+      },
+      "Customer" : {
+        "required" : [ "first_name", "last_name", "email", "personal_id", "phone", "agreements" ],
+        "type" : "object",
+        "properties" : {
+          "first_name" : {
+            "type" : "string",
+            "example" : "Toomas",
+            "description" : "Format: /^([a-zA-ZÄÖÜÕŽŠäöüõžš]+[\\s\\-\\']?)+$/ui",
+            "maxLength": 255
+          },
+          "last_name" : {
+            "type" : "string",
+            "example" : "Kask",
+            "description" : "Format: /^([a-zA-ZÄÖÜÕŽŠäöüõžš]+[\\s\\-\\']?)+$/ui",
+            "maxLength": 255
+          },
+          "personal_id" : {
+            "type" : "string",
+            "description" : "Format: /^[3-6]{1}\\d{10}$/",
+            "example" : "37605030299"
+          },
+          "email" : {
+            "type" : "string",
+            "example" : "martelle.judit@mailinator.com"
+          },
+          "phone" : {
+            "type" : "string",
+            "example" : "+3725101234"
+          },
+          "employer_address" : {
+            "$ref" : "#/components/schemas/EmployerAddress"
+          },
+          "tax_id_number" : {
```

Comment:
I'd describe tax_id_number better as currently I have no idea what this field is about.

Reply:
added description "Your company registration number."